release.py

```diff
@@ -617,6 +617,8 @@ def go(argv):
                     # Need to use JENKINS_NODE_TAG parameter for large memory nodes
                     # since it runs qemu emulation
                     linux_platform_params['JENKINS_NODE_TAG'] = 'linux-' + a + '|| large-memory'
```

Comment:
as I look more closely at the results of the `arm*` debbuilds on native arm machines, I see that many of them failed. It may be safest to require `large-memory` for all `ign-physics` debbuilds, rather than `|| large-memory`

Reply:
I agree.

Reply:
arm native nodes were not able to compile the physics due to the lack of more memory. Is this the case, right?

Reply:
hm...

* focal armhf passed on an ARM native node: [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=ign-physics5-debbuilder&build=380)](https://build.osrfoundation.org/job/ign-physics5-debbuilder/380/) https://build.osrfoundation.org/job/ign-physics5-debbuilder/380/
* focal arm64 ran on r2d2 with qemu anyway: [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=ign-physics5-debbuilder&build=381)](https://build.osrfoundation.org/job/ign-physics5-debbuilder/381/) https://build.osrfoundation.org/job/ign-physics5-debbuilder/381/

but I had big troubles with bionic arm* builds on arm native nodes, though perhaps this is not memory-related:

* [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=ign-physics5-debbuilder&build=392)](https://build.osrfoundation.org/job/ign-physics5-debbuilder/392/) https://build.osrfoundation.org/job/ign-physics5-debbuilder/392/
* [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=ign-physics5-debbuilder&build=393)](https://build.osrfoundation.org/job/ign-physics5-debbuilder/393/) https://build.osrfoundation.org/job/ign-physics5-debbuilder/393/